CriteoPublisherSdk/Sources/DataConsent/CR_Gdpr.h

```diff
@@ -57,6 +57,14 @@ typedef NS_ENUM(NSInteger, CR_GdprTcfVersion) {
 
 - (instancetype)initWithUserDefaults:(NSUserDefaults *)userDefaults NS_DESIGNATED_INITIALIZER;
 
+/**
+ * Gives the status of a User consent for a given purpose
+ *
+ * @param id Purpose id
+ * @return YES if user consent for purpose
+ */
+- (BOOL)consentToPurpose:(NSUInteger)id;
```

Comment:
I find the naming a bit confusing because `consent` is a verb. So it looks like the method will give consent to the given purpose. I imagine something like: `isConsentGivenForPurpose` would be clearer.

Reply:
Done